packages/react-nft-checkout/src/components/PriceConversion/PriceConversion.tsx

```diff
@@ -0,0 +1,79 @@
+import { BN } from '@distributedlab/tools'
+import { Box, Card, Divider, Typography } from '@mui/material'
+import { EstimatedPrice } from '@rarimo/nft-checkout'
+
+import { LoadingIndicator } from '@/components'
+import { useDappContext } from '@/hooks'
+
+type Props = {
+  isLoading: boolean
+  estimatedPrice?: EstimatedPrice
+}
+
+const PriceConversion = ({ isLoading, estimatedPrice }: Props) => {
+  const { targetNft, checkoutOperation } = useDappContext()
+
+  return (
+    <>
+      {isLoading ? (
+        <LoadingIndicator text="Fetching best price" />
+      ) : (
+        estimatedPrice &&
+        targetNft && (
+          <Card
+            sx={{
+              display: 'flex',
+              flexDirection: 'column',
+              gap: 1,
+              mt: 1,
+            }}
+          >
+            <Box
+              display="flex"
+              flexDirection="column"
+              justifyContent="space-between"
+            >
+              <Box
+                display="flex"
+                flexDirection="row"
+                justifyContent="space-between"
+              >
+                <Typography variant="subtitle1">Price conversion</Typography>
+                <Typography variant="subtitle1">
+                  {`${BN.fromBigInt(
+                    targetNft.price.value,
+                    targetNft.price.decimals,
+                  )} ${
```

Comment:
probably not needed parse to BN, try just `targetNft.price` - it implements toString